firedrake/slate/preconditioners.py

```diff
@@ -199,13 +200,17 @@ def initialize(self, pc):
             tensor=self.schur_rhs,
             form_compiler_parameters=self.ctx.fc_params)
 
+        self.mat_type = PETSc.Options().getString(prefix + "S_mat_type", "aij")
```

Comment:
The PC only makes one matrix, I would prefer if this were just called `mat_type`, rather than `S_mat_type`.